### DISCUSSION TITLE
What are the consequences of for companies that unethically share data

### DISCUSSION THREAD

**Discussion by sjgknight** [sjgknight/DSI_collection_sheets#21] [Q&A] [answered]
After reviewing all the consequences for unethical use of data by corporations, it appears that fines are the most common? These are generally token amounts for the corporation so there is very little incentive that discourages unethical sharing of data? What other means are available to prevent companies from sharing info unethically?

**Comment by passiona** [comment, marked as answer]
Internationally, governments have been gradually moving towards higher penalties against corporations that have used or disclosed data that goes beyond the purposes for which the data has been collected. Overall, governments have been more likely to enforce new and existing regulations. 
In the EU the number of fines issues under the GDPR have at least doubled, year on year from 2018 (10 fines) to 2021 (460 fines) (source: https://www.enforcementtracker.com/?insights).
Despite the greater enforcement in the EU, the highest dollar value for privacy breaches have been issued by US regulators. 
Australia is currently pursuing an action against Facebook in the Federal Court for privacy breaches related to Cambridge Analytics, and if successful will likely result in a fine. Australia has also drafted an amendment to its Privacy Act raising the maximum penalty from AUD$2.1M to AUD$10M.
These are all examples of deterrence.  From the consumer's perspective, to _prevent_ the sharing of data, there are a few different approaches. One is to select corporations you deal with based on their privacy protection track record and some corporations make this a point of difference. Another approach is to minimise the data available on you and this could include adjusting settings on your devices to even utilising cash payments for certain items to minimise transaction tracing. A final approach, and this may only be relevant in some circumstances is to provide data that is unique,  for example, our Privacy Act provides for example the right to use a pseudonym instead of your legal name, and previous surveys conducted in Australia have revealed that, particularly in online settings where users are forced to provide information to access a service, that false information is sometimes provided by consumers. I have a colleague that provides slightly different details to different services, in order to keep track of where data may have been leaked from.
